### DISCUSSION TITLE
Renovate only changes pnpm-lock.yaml though I don't want it. Why?

### DISCUSSION THREAD

**Discussion by onissen** [renovatebot/renovate#33320] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc.) and which version of Renovate.

GitHub App on cloud

### Please tell us more about your question or problem

I am very new to using the renovate app on GitHub. I am using renovate in a monorepo powered with turborepo. The onboarding PR found every `package.json` right.

When crating a Pull Request with a dependency update, Renovate only changes the `pnpm-lock.yaml` but not the `package.json` where the usage of the updated dependency is defined in. I cannot find the reason for this behavior since I did not intentional configure renovate like this.

See my `renovate.json`:

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:recommended"],
  "semanticCommits": "enabled",
  "schedule": ["* 13-20 * * 5,6"],
  "labels": ["dependencies"],
  "reviewers": ["onissen"],
  "timezone": "Europe/Amsterdam"
}
```

Before I started using renovate, I was using dependabot for version updates. When I merged the onboarding PR it was activated through the `.github/dependabot.yaml`. May this be a reason for the behavior? I have removed this configuration file afterwards.
See my current settings for code security on GitHub:

![image](https://github.com/user-attachments/assets/260c8184-a438-46c7-941a-b1e4e5274ec5)

Can you see how I can make renovate to update `package.json` as well as `pnpm-lock.yaml`?

Thanks for your help.

### Logs (if relevant)

See the log of this example PR: https://github.com/ncs-northware/northware/pull/239

I am sorry that I must give you the full log: https://gist.github.com/onissen/fd2b6a9d5e8c5232d6e40ed751b4265d
I have created a couple of gists. One where I tried to extract relevant contents from my PR, one with the full log where this PR was created and one where the PR was rebased.

**Comment by rarkins** [comment, marked as answer]
Read docs on rangeStrategy

**Reply by onissen** [reply]
Thanks for the tip @rarkins, not it is working.
